.github/CODEOWNERS

```diff
@@ -5247,6 +5247,7 @@
 /types/react-stripe-elements/                           @dan-j @santiagodoldan @sonnysangha @9y5 @thchia @yhnavein @virzak @remotealex @bombek92 @hirochachacha @paustint @mastacheata
 /types/react-svg-inline/                                @kiyopikko
 /types/react-svg-pan-zoom/                              @huy-nguyen
+/types/react-svg-radar-chart/                           @luksys5
```

Comment:
I believe this is automatic, not changed manually via PR
```suggestion
```

Reply:
Will remove then this one. Thanks